bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -122,297 +109,175 @@ If you did not manually create things in the *.things file, the Netatmo Binding
-| DateMaxTempThisMonth| DateTime             | Date when maximum temperature was reached this month     |
-| TimeStamp           | DateTime             | Timestamp when data was measured                         |
-| LastMessage         | DateTime             | Last message emitted by the module                       |
-| LowBattery          | Switch               | Low battery                                              |
-| BatteryVP           | Number               | Battery level                                            |
-| RfStatus            | Number               | Signal strength (0 for no signal, 1 for weak, 2 for average, 3 for good or 4 for excellent) |
+| Channel Group       | Channel Id           | Item Type            | Description                                      |
+|---------------------|----------------------|----------------------|--------------------------------------------------|
+| humidity            | value                | Number:Dimensionless | Current humidity                                 |
+| humidity            | humidex              | Number               | Computed Humidex index                           |
+| humidity            | humidex-scale        | Number               | Humidex index appreciation                       |
+| humidity            | heat-index           | Number:Temperature   | Computed Heat Index                              |
+| humidity            | dewpoint             | Number:Temperature   | Computed dewpoint temperature                    |
+| humidity            | dew-point-depression | Number:Temperature   | Computed dewpoint depression                     |
+| temperature         | value                | Number:Temperature   | Current temperature                              |
+| temperature         | today-min            | Number:Temperature   | Minimum temperature on current day               |
+| temperature         | today-max            | Number:Temperature   | Maximum temperature on current day               |
+| temperature         | min-today            | Number:Temperature   | Timestamp of today's minimum temperature         |
+| temperature         | max_today            | Number:Temperature   | Timestamp of today's maximum temperature         |
+| temperature         | trend                | String               | Temperature evolution trend over time            |
+| timestamp           | last-seen            | DateTime             | Timestamp when data was measured                 |
+| signal              | strength             | Number               | Signal strength (0 for no signal, 1 for weak...) |
+| signal              | value                | Number:Power         | Signal strength in dB                            |
+| battery             | value                | Number               | Battery level                                    |
+| battery             | low-battery          | Switch               | Low battery                                      |
```

Comment:
There is also a `status` channel of type String.

Reply:
Added